config/sync/core.entity_view_display.entity_subqueue.discover_topics.default.yml

```diff
@@ -26,7 +26,7 @@ content:
       field_formatter_range:
         limit: 0
         offset: 0
-        reverse: false
+        order: 0
```

Comment:
In the upstream version of field_formatter_range, between version 1.1 and 1.2, there was a BC change that caused issues with our configuration. The change is that they removed the property reverse (boolean) and replaced it with the property order (integer).